internal/tiltfile/links/links.go

```diff
@@ -117,6 +101,10 @@ func (e Extension) link(thread *starlark.Thread, fn *starlark.Builtin, args star
 	}
 
 	return Link{
+		Struct: starlarkstruct.FromStringDict(starlarkstruct.Default, starlark.StringDict{
```

Comment:
```suggestion
		Struct: starlarkstruct.FromStringDict(starlark.String("link"), starlark.StringDict{
```

I don't know why the code calls the first arg "constructor", but it looks like it acts as the type. It shows up in `.String()` and is compared when checking if two structs are equal or can be added (apparently structs support addition!?)

Reply:
done